frontend/src/app/services/event-list.ts

```diff
@@ -31,6 +30,30 @@ function probability(probabilityRate: number){
   return probabilityRate > Math.random();
 }
 
+/**
+ * Returns true if EventMitigation is active
+ * @param eventInput - EventInput
+ * @param id - Id of the EventMitigation
+ */
+function isEventMitigationActive(eventInput: EventInput, id: string) {
+  const mitigation = eventInput.eventMitigations.find(em => em.id === id);
+  return !isNil(mitigation);
+}
+
+/**
+ * Callback that is called every day before trigger conditions are evaluated
+ * @param eventInput - EventInput
+ * @param id - Id of the EventMitigation
+ */
+export function eventStateUpdate(eventInput: EventInput) {
```

Comment:
What do you need this function for? Are you using it for event emission anywhere?
If this is a preparation for future, I would stay with stat i.e. extending them.

Reply:
This is a place where we can calculate event specific stats, e.g. the number of days that the number of cases was more than x.

Reply:
Yeah, I see it. Why not extending the stats? I would keep the language (data structures, function set) as minimalistic as possible. We have stats, we can easily extend them, it's a transparent structure.

Reply:
No problem. Ok to still call the extended structure "EventInput"?